src/main/lairKeystore.ts

```diff
@@ -62,26 +80,35 @@ export async function launchLairKeystore(
       return Promise.reject(`Failed to create symlinked lair directory: ${e}`);
```

Comment:
This could maybe become a TRPCError as well then?

Reply:
fiexd but I believe we can simplify it further by utilizing tRPC's full capabilities. Instead of isolating event listening in a separate function, we could use tRPC subscriptions for status updates and handle actions through mutations.

This approach would streamline our code by separating real-time data handling from the main logic, making it cleaner and more maintainable. It aligns better with tRPC best practices and could enhance performance

Reply:
but lets work step by step